tests/fixtures/testrepo/upstream_with_epoch/repoclosure/el7.conf

```diff
@@ -0,0 +1,33 @@
+[main]
```

Comment:
do we need this file, or is it here because we have it in the other dummy repos too? I'd (personally) drop it, but would not force you to :)